Yosemite/YosemiteTests/Stores/ProductVariationStoreTests.swift

```diff
@@ -455,7 +455,7 @@ final class ProductVariationStoreTests: XCTestCase {
 
     /// Verifies that `ProductVariationAction.updateProductVariation` returns an error whenever there is an error response from the backend.
     ///
-    func testUpdatingProductVariationReturnsErrorUponReponseError() {
+    func testUpdatingProductVariationReturnsErrorUponResponseError() {
```

Comment:
Do it worth renaming the method using the underscore?

Reply:
yup updated in: https://github.com/woocommerce/woocommerce-ios/pull/3278/commits/f436d3e9aefd1ff68f23e1cc89639c70b39fd44f